sdk/dotnet/AdmissionRegistration/V1/Inputs/MutatingWebhookConfigurationArgs.cs

```diff
@@ -0,0 +1,12 @@
+// *** WARNING: this file was generated by pulumigen. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Kubernetes.Types.Inputs.AdmissionRegistration.V1
```

Comment:
Is this namespace needed? Not sure if it would be possible to omit the file if the type doesn't have args?

Reply:
It's not needed but does no harm other than polluting the codebase a bit. It will go away when we switch the compat mode off, so I decided not to bother. Do you feel we should remove it?

Reply:
No, not a big deal.